NitroxServer/Communication/UdpServer.cs

```diff
@@ -126,21 +131,21 @@ private void ConnectionStatusChanged(NetConnectionStatus status, NetConnection n
             else if (status == NetConnectionStatus.Disconnected)
             {
                 Connection connection = GetConnection(networkConnection.RemoteUniqueIdentifier);
-                Player player = playerManager.GetPlayer(connection);
+                Player player = NitroxServiceLocator.LocateService<PlayerManager>().GetPlayer(connection);
```

Comment:
PlayerManager is already passed to this function. 

In general, NitroxServiceLocator is meant to initialize objects with dependencies that are also registered in the same AutoFac container. Using it as a static singleton service like here defeats the benefits of using AutoFac.